.goreleaser.yml

```diff
@@ -110,20 +110,18 @@ brews:
       system "#{bin}/terraform-ls --version"
 
 publishers:
-  - name: "hc-releases"
+  - name: upload
     checksum: true
     signature: true
     env:
-      - AWS_DEFAULT_REGION={{ .Env.AWS_DEFAULT_REGION }}
-      - AWS_REGION={{ .Env.AWS_REGION }}
-      - AWS_ACCESS_KEY_ID={{ .Env.AWS_ACCESS_KEY_ID }}
-      - AWS_SECRET_ACCESS_KEY={{ .Env.AWS_SECRET_ACCESS_KEY }}
-      - AWS_SESSION_TOKEN={{ .Env.AWS_SESSION_TOKEN }}
-    cmd: hc-releases upload-file {{ abs .ArtifactPath }}
-
 milestones:
   - name_template: "{{ .Tag }}"
     close: true
+      - HC_RELEASES_HOST={{ .Env.HC_RELEASES_HOST_STAGING }}
+      - HC_RELEASES_KEY={{ .Env.HC_RELEASES_KEY_STAGING }}
+    dir: "{{ dir .ArtifactPath }}"
+    cmd: |
+      hc-releases upload -file={{ .ArtifactName }} -product={{ .ProjectName }} -version={{ .Version }}
```

Comment:
Before this change, we passed a directory containing all release artifacts to `hc-releases`, `hc-releases` uploaded all files from this directory to the S3 bucket.

After the change, we're passing only a single file as an argument to `hc-releases`. Will this work for a release with multiple files?

Reply:
The new `hc-releases` has a `-file` parameter that can be specified multiple times.  So in general, it can be called or multiple times to upload, whatever best fits the workflow.  In this particular case, I think goreleaser is running the hook once per artifact, no?

Reply:
@dbanck after some testing, we've changed it from`-file={{ .ArtifactName }} ` to `-file={{ .ArtifactPath }}` to allow multiple files.